java/integTest/src/test/java/glide/LoggerTests.java

```diff
@@ -72,9 +72,6 @@ public void log_to_file() {
         // Initialize a new logger to force closing of existing files
         String dummyFilename = "dummy.txt";
         Logger.setLoggerConfig(Logger.Level.DEFAULT, dummyFilename);
```

Comment:
If we are removing all dummy log related tests, why not removing these 2 lines as well?